src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Delete/ConditionalDeleteResourceHandler.cs

```diff
@@ -86,33 +92,47 @@ public async Task<DeleteResourceResponse> Handle(ConditionalDeleteResourceReques
 
         private async Task<DeleteResourceResponse> DeleteSingleAsync(ConditionalDeleteResourceRequest request, CancellationToken cancellationToken)
         {
-            var matchedResults = await _searchService.ConditionalSearchAsync(
+            var results = await _searchService.ConditionalSearchAsync(
```

Comment:
What would happen if a search parameter was disabled for the link requested by the bulk delete job? Should we return a warning to the caller / on the job response?

Reply:
You mean an unsupported search parameter? There is a check to look for those errors, it fails the call/job with a bad request if any are found.

Reply:
Then this scenario should be covered. If a customer tries to use a disabled search param, this will return an error.